runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemInputListener.java

```diff
@@ -50,7 +57,22 @@ public void keyPressed(KeyEvent e)
 	{
 		if (e.getKeyCode() == HOTKEY)
 		{
-			plugin.setHotKeyPressed(true);
+			if (plugin.isHideAll())
+			{
+				plugin.setHideAll(false);
+				plugin.setHotKeyPressed(true);
+				lastPress = null;
+			}
+			else if (lastPress != null && !plugin.isHotKeyPressed() && config.doubleTapDelay() > 0 && Duration.between(lastPress, Instant.now()).compareTo(Duration.ofMillis(config.doubleTapDelay())) < 0)
```

Comment:
Alternatively, the explicit check for `> 0` could be removed, since there's no way to register two presses in that time.

Reply:
> Alternatively, the explicit check for `> 0` could be removed, since there's no way to register two presses in that time.

Yeah not sure whether it's necessary, but thought it might be good practice to explicitly check.